sql/telemetry_derived/clients_histogram_aggregates_v1/init.sql

```diff
@@ -19,5 +20,12 @@ CREATE TABLE IF NOT EXISTS
       aggregates ARRAY<STRUCT<key STRING, value INT64>>
     >>
 )
-PARTITION BY RANGE_BUCKET(sample_id, GENERATE_ARRAY(0, 100, 1))
-CLUSTER BY app_version, channel, client_id
+PARTITION BY submission_date
+CLUSTER BY sample_id, app_version, channel
+OPTIONS(require_partition_filter=true);
```

Comment:
If a new table is going to be created, the `partition_expiration_days=3` option can probably be moved into this section.

Reply:
Yes, I'll be creating a new table, so I'll move this into the existing options.